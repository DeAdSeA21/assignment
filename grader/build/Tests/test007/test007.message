Output line 14:
  <i>expected: </i><span class='expected'>13</span>\t1010.89
  <i>observed: </i>-<span class='observed'>-</span>- end of output ---
