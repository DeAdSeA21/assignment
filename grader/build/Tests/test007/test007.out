Starting loan balance: Annual interest rate: Monthly payment: 
Month	Balance
1	1000.83
2	1001.67
3	1002.50
4	1003.34
5	1004.17
6	1005.01
7	1005.85
8	1006.69
9	1007.53
10	1008.36
11	1009.20
12	1010.05
